js/components/discussions/thread.vue

```diff
@@ -148,7 +151,11 @@ export default {
             } else {
                 this.$scrollTo(this);
             }
+        },
+        formatDate(val) {
+            return moment(val).format('LL')
```

Comment:
Unfortunately as far as I'm concerned, we're not using StandardJS conventions, so I think we need a `;` here.